nixos/modules/services/networking/suricata/settings.nix

```diff
@@ -603,12 +702,14 @@ in
     "reference-config-file" = mkOption {
       type = types.str;
       default = "${cfg.package}/etc/suricata/reference.config";
+      defaultText = "/etc/suricata/reference.config";
```

Comment:
defaultText should represent the default, not something else. Please try `defaultText = "\${config.services.suricata.package}/etc/suricata/reference.config"`.